test/unit/coffee/PackManager/PackManagerTests.coffee

```diff
@@ -20,6 +20,7 @@ describe "PackManager", ->
 			"./LockManager" : {}
 			"./MongoAWS": {}
 			"logger-sharelatex": { log: sinon.stub(), error: sinon.stub() }
+			'./Metrics': {inc: ()->}
```

Comment:
Hmm, looks like we have this wrapper file here too. Let's clean this up and just use `metrics-sharelatex` as part of this.